mypy/messages.py

```diff
@@ -6,7 +6,7 @@
 import re
 import difflib
 
-from typing import cast, List, Dict, Any, Sequence, Iterable, Tuple
+from typing import cast, List, Dict, Any, Sequence, Iterable, Tuple, Optional
```

Comment:
Good catch!
I changed my PyCharm settings to complain if there's an unused import when I'm committing.